### PR TITLE
Topix fix

### PR DESCRIPTION
**Pull request by maaikelimper**
this combines a fix of the topic structure
https://github.com/wmo-im/wis2box/issues/244
with additional documentation in quickstart to guide the user to setup his/her specific dataset
https://github.com/wmo-im/wis2box/issues/246

### CODE REVIEW

File: config_examples/data-mappings.yml.example-synop-bufr [hunk 1]
```diff
@@ -0,0 +1,17 @@
+data:
+    member_code3.center_id.data.core.weather.surface-based-observations.SYNOP:
```
Comment (david-i-berry) [inline config_examples/data-mappings.yml.example-synop-bufr:2]: I would make clear that member_code3 is the ISO 3166 alpha-3 code.  Maybe call it ISO3C_country or similar.

https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3

File: config_examples/dev.env.example [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# please define a data-directory on your host machine
+# this will map to /data/wis2box on the wis2box-container
+WIS2BOX_HOST_DATADIR=your/data/directory
+
+# Optional
+# Environment variable overrides
+WIS2BOX_LOGGING_LOGLEVEL=WARNING
+WIS2BOX_DATA_RETENTION_DAYS=30
+
+WIS2BOX_DATADIR_DATA_MAPPINGS=/data/wis2box/data-mappings.yml
```
Comment (david-i-berry) [inline config_examples/dev.env.example:10]: This needs to go in the Docker default.env file.

File: docs/source/quickstart.rst [hunk 3]
```diff
@@ -3,76 +3,169 @@
...
     git clone https://github.com/wmo-im/wis2box.git
     cd wis2box
 
 
-For the purposes of a quickstart, this deployment expects the test environment, which includes data and metadata, and runs on localhost. This
-is done by using the test environment file:
+wis2box configuration
+---------------------
+
+Wis2box will read environment variables from dev.env. A baseline is provided in dev.env.example. Copy dev.env.example to dev.env
```
Comment (david-i-berry) [inline docs/source/quickstart.rst:42]: This is actually wrong, the majority of the environment variables are defined in the docker/default.env file. This includes specifying the location of the mapping file.

File: docs/source/quickstart.rst [hunk 4]
```diff
@@ -3,76 +3,169 @@
...
 
 .. code-block:: bash
 
-    cp tests/test.env dev.env
-    vi dev.env
+    cp config_examples/dev.env.example dev.env
 
+And update it to suit your needs. *You must replace '/your/data/directory'* with a valid directory or your host.
```
Comment (david-i-berry) [inline docs/source/quickstart.rst:48]: on your host, not or your host.

**Review by david-i-berry — COMMENTED** [review]
Most of the changes look to be to the topic hierarchy, I haven't checked these in detail.

I've added a few comments to the quickstart where some changes are required, mostly around the default.env file. I can propose changes but this won't be until Monday.

File: docs/source/quickstart.rst (hunk 3 above)
Comment (maaikelimper) [reply docs/source/quickstart.rst:42]: the docker-compose will overwrite values defined docker/default.env with those defined in dev.env. So the user should edit them in dev.env.

File: config_examples/dev.env.example (hunk 2 above)
Comment (maaikelimper) [reply config_examples/dev.env.example:10]: Yes they are originally defined in docker/default.env, by redefining them in dev.env the user overrides them. this is how the environment variables in docker-compose currently are implemented. Could be done differently, these instructions are to guide the user to work with how it is done now...

File: config_examples/data-mappings.yml.example-synop-bufr (hunk 1 above)
Comment (maaikelimper) [reply config_examples/data-mappings.yml.example-synop-bufr:2]: ok I've changed this according to your suggestion

File: config_examples/dev.env.example (hunk 2 above)
Comment (maaikelimper) [reply config_examples/dev.env.example:10]: could you check my proposal for baseline data-mappings ? I was wondering, the bufr-example is not actually synop specific is it ?

File: docs/source/quickstart.rst (hunk 4 above)
Comment (maaikelimper) [reply docs/source/quickstart.rst]: fixed

File: config_examples/station_list.csv.example [hunk 5]
```diff
@@ -0,0 +1,5 @@
+station_name,wigos_station_identifier,traditional_station_identifier
+Balaka,0-454-2-AWSBALAKA,AWSBALAKA
```
Comment (david-i-berry) [inline config_examples/station_list.csv.example:2]: Balaka and Malomo do not have traditional station identifiers, these should be left blank.

File: docs/source/quickstart.rst [hunk 6]
```diff
@@ -3,76 +3,178 @@
...
+    python3 wis2box-ctl.py build
 
-    wis2box api add-collection --topic-hierarchy data.core.observations-surface-land.mw.FWCL.landFixed $WIS2BOX_DATADIR/metadata/discovery/mw-surface-weather-observations.yml
+This might take a while. 
 
+wis2box start
+-------------
 
-Cache and publish station collection and discovery metadata to the API:
+Start wis2box with Docker Compose and login to the wis2box container:
```
Comment (david-i-berry) [inline docs/source/quickstart.rst:106]: As written I would wonder how to start the box using Docker Compose (I would be looking for a docker-compose up). What about

"Start wis2box with the wis2box command line tool and log in to the wis2box container. Note, this calls docker-compose in the background."

**Review by david-i-berry — COMMENTED** [review]
One last set of minor comments on the quick start.

**Review by david-i-berry — APPROVED** [review]

**Pull request merged**